### PR TITLE
tmuxinator: Build a bottle for Linuxbrew

### PR DESCRIPTION
**Pull request by dawidd6**
This is an automated pull request to build a new bottle for linuxbrew-core
based on the existing bottle block from homebrew-core.

### CODE REVIEW

**Comment by dawidd6** [comment]
```
==> brew test tmuxinator --verbose
Testing tmuxinator
==> /home/linuxbrew/.linuxbrew/Cellar/tmuxinator/1.1.1/bin/tmuxinator version
==> source /home/linuxbrew/.linuxbrew/Cellar/tmuxinator/1.1.1/etc/bash_completion.d/tmuxinator && complete -p tmuxinator
sh: 1: source: not found
Error: tmuxinator: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<127>.
```

File: Formula/tmuxinator.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def install
     version_output = shell_output("#{bin}/tmuxinator version")
     assert_match "tmuxinator #{version}", version_output
 
-    completion = shell_output("source #{bash_completion}/tmuxinator && complete -p tmuxinator")
+    completion = shell_output("bash -c \"source #{bash_completion}/tmuxinator && complete -p tmuxinator\"")
```
Comment (iMichka) [inline Formula/tmuxinator.rb]: Please split this in a separate command for mac/linux. We do want to keep track what is a linux-only change for readability.

Comment (dawidd6) [reply Formula/tmuxinator.rb]: I mean, this should be submitted upstream.

Comment (dawidd6) [reply Formula/tmuxinator.rb]: Dunno if what I did is what you meant.

**Pull request closed without merging**